src/sagemaker/chainer/estimator.py

```diff
@@ -90,6 +90,10 @@ def __init__(self, entry_point, use_mpi=None, num_processes=None, process_slots_
 
         super(Chainer, self).__init__(entry_point, source_dir, hyperparameters,
                                       image_name=image_name, **kwargs)
+
+        if py_version == 'py2':
+            logger.warning('chainer py2 container will be deprecated soon.')
```

Comment:
* s/chainer py2 container/The Python 2 Chainer images
  * same applies to the others (also make sure the right framework name is in the message for some of the ones below)
* I'd add a more actionable statement after the warning, e.g. "Set py_version='py3' to use a Python 3 image." This would be especially helpful with the frameworks where Python 2 is the default - users may not be aware of the `py_version` kwarg when using those.